lib/reference.coffee

```diff
@@ -160,6 +172,7 @@ class Reference
     for i in [1...bookId]
       count += Reference.chaptersInBookId(i)
     return count
+    return 0
```

Comment:
Yes, I put that there while trying to figure out why I was getting a certain result. Taken out.